components/molecules/MarkdownWrapper/markdown-wrapper.tsx

```diff
@@ -0,0 +1,39 @@
+import ReactMarkdown from "react-markdown";
+import SyntaxHighlighter from "react-syntax-highlighter";
+import { atomOneDark } from "react-syntax-highlighter/dist/cjs/styles/hljs";
+
+interface MarkdownWrapperProps {
+  content: string;
+  pClassName?: string;
+}
+
+const MarkdownWrapper = ({ content, pClassName }: MarkdownWrapperProps) => (
+  <ReactMarkdown
+    // eslint-disable-next-line react/no-children-prop
+    children={content}
+    className=""
```

Comment:
nit: if we are not currently passing any class here, we can remove this 👍

Reply:
Thank you! I meant to remove that.